src/applications/vaos/components/calendar/CalendarRow.jsx

```diff
@@ -85,49 +85,56 @@ export default class CalendarRow extends Component {
         );
 
         return (
-          <div className={cssClasses}>
-            {optionsError && (
-              <span
-                className="usa-input-error-message vads-u-margin-bottom--2 vads-u-padding-top--0 vads-u-width--full"
-                role="alert"
-              >
-                <span className="sr-only">Error</span> {optionsError}
-              </span>
-            )}
-            {selectedDateOptions.map((o, index) => {
-              const dateObj = {
-                date: currentlySelectedDate,
-                [fieldName]: o.value,
-              };
-              const checked = isDateOptionPairInSelectedArray(
-                dateObj,
-                selectedDates,
-                fieldName,
-              );
+          <fieldset>
+            <legend
```

Comment:
The legend needs to be refactored a bit, because current markup makes it invisible to screen readers:

```html
<legend class="vads-u-visibility--screen-reader">
  {additionalOptions.legend}
</legend>
```